docs/tactics.md

```diff
@@ -1138,10 +1138,30 @@ Here too, the `reassoc` attribute can be used instead. It works well when combin
 attribute [simp, reassoc] some_class.bar
 ```
 ### sanity_check
+User commands to spot common mistakes in the code
 
-The `#sanity_check` command checks for common mistakes in the current file or in all of mathlib, respectively.
+* `#sanity_check`: check all declarations in the current file
+* `#sanity_check_mathlib`: check all declarations in mathlib (so excluding core or other projects,
+  and also excluding the current file)
+* `#sanity_check_all`: check all declarations in the environment (the current file and all
+  imported files)
 
-Currently this will check for unused arguments in declarations and whether a declaration is incorrectly marked as a def/lemma.
+You can append a `-` to any command (e.g. `#sanity_check_mathlib-`) to only run the subset of the
+tests that don't take long to execute.
+
+Currently this will check for
+
+* unused arguments in declarations,
+* whether a declaration is incorrectly marked as a def/lemma,
+* whether a namespace is duplicated in the name of a declaration
+* whether ≥/> is used in the declaration
+
```

Comment:
Perhaps these items should be numbered, and the comment about `-` should be moved below and mention by number which ones are run by `sanity_check-`. Right now it isn't clear which ones take long to execute.

Reply:
good idea!